pkg/testframework/imageregistry.go

```diff
@@ -342,3 +342,24 @@ func MustEnsureClusterOperatorStatusIsSet(t *testing.T, client *Clientset) {
 		t.Fatal(err)
 	}
 }
+
+func MustEnsureOperatorIsNotHotLooping(t *testing.T, client *Clientset) {
+	// Allow the operator a few seconds to stabilize
+	time.Sleep(15 * time.Second)
```

Comment:
This function is called after MustEnsureImageRegistryIsAvailable, which waits for Progressing == False && Available == True. Doesn't that condition mean that the operator has done its job and now it's in the stable state?

Reply:
The registry can be in the available state and there still be a looping issue in say, syncStorage or syncSecrets.

Reply:
yeah, the hotlooping condition we've seen doesn't cause the registry itself to redeploy, it just causes us to keep updating the CR repeatedly, which in turn triggers all the resource rendering logic.  The generated resources don't change (hence no re-deploy) but it hammers the api server.